scripts/dockerimages/scripts/console.sh

```diff
@@ -74,6 +74,18 @@ EOF
 if ! grep -q '^UseDNS no' /etc/ssh/sshd_config; then
     echo "UseDNS no" >> /etc/ssh/sshd_config
 fi
+cat > /etc/os-release << EOF
+NAME="Rancher OS"
+VERSION=2.1
```

Comment:
let's not hard code this.  Can you pull this from `rancherctl -v | awk '{print $NF}'`